src/amqp10/encoder.ts

```diff
@@ -47,7 +47,7 @@ type MessageAnnotationsList = { key: string; value: MessageAnnotationsValue }[]
 
 export function amqpEncode(
   writer: DataWriter,
-  { content, messageProperties, applicationProperties, messageAnnotations }: Message
+  { content, messageProperties, applicationProperties, messageAnnotations }: Message,
```

Comment:
If you set a property in the prettierc 
```
  "trailingComma": "es5"
```
You don't have to add this orrible comma and you don't have to change all these files